app/src/test/java/org/simple/clinic/selectstate/SelectStateUiRendererTest.kt

```diff
@@ -43,16 +44,7 @@ class SelectStateUiRendererTest {
     // then
     verify(ui).showStates(states = states, selectedState = andhraPradesh)
     verify(ui).showNextButton()
-    verifyNoMoreInteractions(ui)
-  }
-
-  @Test
-  fun `when states are not fetched, then hide the states list`() {
```

Comment:
Why was this test removed? 🤔

Reply:
Basically, this is the same as states fetch error. So removed this.